src/graphql/main/schema.graphql

```diff
@@ -1187,6 +1187,20 @@ type Query {
   userDefaultWalletId(username: Username!): WalletId!
     @deprecated(reason: "will be migrated to AccountDefaultWalletId")
   usernameAvailable(username: Username!): Boolean
+  directReferralLeaderboard(
+    range: ReferralRange!
+    after: String
+    before: String
```

Comment:
Do we need pagination? Should we not just show only the first xxx entries anyhow (maybe 100?)

Reply:
A user may potentially want to see who is directly above and below them